### ISSUE TITLE
Simple Tab Groups extension doesn't display properly on Waterfox.

### ISSUE THREAD

**Issue by brdigetrlol** [MrAlex94/Waterfox#3012]
### What happened?

The Simple Tab Groups extension for Firefox doesn't work on Waterfox. I'm not really sure where the issue might be, but I've attached a screenshot showing that settings box doesn't display properly on Waterfox (it works just fine with Firefox).

I know it's an issue with a single extension, but I'm assuming that there must be some implications for other extensions or elsewhere in the browser.

![image](https://user-images.githubusercontent.com/7166950/231897535-d8190e37-000f-4459-b28f-90168540eb01.png)

### Reproducible?

- [X] I have tested this on Firefox.

### Version

G5

### What platform are you seeing the problem on?

Windows

### Relevant log output

```shell
15:40:03.691 BG onFocusChangedWindow 1 (onFocusChangedWindow@stg-background.js:917:34) logger.js:204:23
15:40:03.692 Menus.update ▷ set-tab-icon-as-group-icon 
Object { enabled: true }
 (update@js/menus.js:81:24 ◁ onFocusChangedWindow@stg-background.js:920:15) logger.js:204:23
15:40:03.704 Menus.update ◁ (update@js/menus.js:90:9) logger.js:204:23
15:40:07.541 TypeError: this.isEnabled is not a function 2 logger.js:183:15
15:40:10.069 BG onFocusChangedWindow -1 (onFocusChangedWindow@stg-background.js:917:34) logger.js:204:23
```

**Comment by lazymonkey2** [comment]
Works for me on Waterfox 5.1.5
I suggest to create a new profile and try again.

![image](https://user-images.githubusercontent.com/26486758/235372332-7f2d3876-4274-4cb4-b7ea-5252f909755e.png)

**Comment by brdigetrlol** [comment]
Yeah, it's actually working now and I didn't even do anything that I can remember...

Weird. Anyway, thank you!
